README.md

```diff
@@ -267,6 +268,7 @@ In any case the current scale is not considered, so your system won't change if
 |`setAllowFlingInOverscroll(boolean)`|If true, fling gestures will be allowed even when detected while overscrolled. This might cause artifacts so it is disabled by default.|`false`|
 |`panTo(float, float, boolean)`|Pans to the given values, animating if needed.|`-`|
 |`panBy(float, float, boolean)`|Applies the given deltas to the current pan, animating if needed.|`-`|
+|`cancelAnimations()`|Cancels all currently active code driven animations (including fling animations).|`-`|
```

Comment:
I would say a finger fling is not code driven?

Reply:
Well the user isn't touching the screen anymore so... I would argue it is :smile: but I see your point.
Do you have a suggestion? Would you just write:
```
Cancels all currently active animations.
```
?

Reply:
I would be more specific and say like, animations triggered by API calls with `animate = true` and fling animations triggered by touch input.

Reply:
@natario1 Ok with the current vesion?

Reply:
Looks good to me! @markusressel